app/webpacker/components/identity-verification-form.js

```diff
@@ -0,0 +1,38 @@
+class IdentityVerificationForm {
+  constructor() {
+    this.formElt = document.querySelector('.js-identity-verification-form');
+    if (!this.formElt) return
+
+    this.inputElts = this.formElt.querySelectorAll('input[type="text"]')
+    this.inputElts[0].focus()
+    this.inputElts.forEach(elt => elt.addEventListener("input", this.onInputChange))
+    this.inputElts.forEach(elt => elt.addEventListener("keyup", this.onKeyUpChange))
+    this.inputElts.forEach(elt => elt.addEventListener("keydown", this.onKeyDownChange))
+  }
+
+  onInputChange = (event) => {
+    const inputElt = event.currentTarget
+    inputElt.value = inputElt.value.toUpperCase()
+    const idx = [...this.inputElts].indexOf(inputElt)
+    if (idx === 2) {
```

Comment:
On peut plutôt vérifier que `inputElt` est le dernier élément de `this.inputElts`, plutôt que de se baser sur l’index? Comme ça, ça ne dépend pas du fait qu’on demande 3 lettres.